### PR TITLE
Rebase modupdater

### PR DESCRIPTION
**Pull request by SuperSandro2000**
Based on work from https://github.com/ALovedOne and https://github.com/patschi

Needs proper testing!

### CODE REVIEW

**Comment by carlbennett** [comment]
I think these changes broke the latest version of the image. See #268 and #269

**Comment by carlbennett** [comment]
The .sh files added in this change are not marked executable (`chmod +x`).

File: 0.17/files/docker-entrypoint.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/bin/sh -x
```
Comment (carlbennett) [inline 0.17/files/docker-entrypoint.sh]: This also left the image in an un-runnable state.

Comment (SuperSandro2000) [reply 0.17/files/docker-entrypoint.sh]: Probably need to use ash or I just install bash.

**Comment by SuperSandro2000** [comment]
@Fank If I write that it needs proper testing then I mean I did not test it properly and it shouldn't be merged. We need to communicate this better next time.

**Comment by Fank** [comment]
I thought people were using it so I was thinking it's maybe bugged with mod, and not making it unstable

**Comment by SuperSandro2000** [comment]
I should have written in the description that I changed it slightly. Next time we do it better.

**Comment by carlbennett** [comment]
In my report #268, I don't have any extra mods loaded, just the base mod from Factorio that you can't disable. It was breaking on new setups too.

**Comment by SuperSandro2000** [comment]
I will take another look at it later today and fix it.

**Comment by SuperSandro2000** [comment]
@carlbennett Can't reproduce that. Did you pull the latest image?

**Comment by carlbennett** [comment]
@SuperSandro2000 My fixes from #272 fixed the problem. It's been fixed since yesterday. I was commenting in reply to @Fank's comment:
>I thought people were using it so I was thinking it's maybe bugged with mod, and not making it unstable

I had no mods loaded besides the base mod that you have to load.

There is no present bug in the latest head on master (6ae5795d383d9e9a38fac55123a189cfa233f7c6).

**Pull request merged**